package.json

```diff
@@ -1,5 +1,8 @@
 {
   "name": "community_compose",
   "private": true,
-  "dependencies": {}
+  "dependencies": {
+    "fetch-rails": "^0.8.3",
```

Comment:
these are the javascript packages I installed to make the edit functionality work